### DISCUSSION TITLE
Allow query to run when window is opened in a new tab (not focused)

### DISCUSSION THREAD

**Discussion by rohmann** [TanStack/query#1362] [Q&A]
I'm working on an application where it's common for a user to open multiple documents in new tabs. I'm also using the query result to set the `title` tag. What I'm observing is the query remains disabled until the first time I actually focus the browser tab, so the user doesn't see the correct document title in their browser tabs until they've viewed each tab at least once which isn't ideal.

Is there a way to ensure certain query always run even if the browser window isn't in focus?

**Comment by TkDodo** [comment]
I think that v3 exposes a `focusManager`:

https://react-query-beta.tanstack.com/guides/window-focus-refetching#managing-focus-state

by calling `focusManager.setFocus(true)` when the page opens in the new tab, it basically puts react-query in the "focused" state, which means react-query should run as if it's the currently focused tab.

I'm just guessing here 😅 but maybe you could try that :)
